src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -222,14 +221,18 @@ public static String getIndividualFullName(Entity entity) {
         return entity.getName().getFullNameNoTitle();
     }
 
-    public static String getIndividualInterpreterLanguage(HearingOptions hearingOptions, OverrideFields overrideFields, ReferenceDataServiceHolder referenceData) throws InvalidMappingException {
+    public static String getIndividualInterpreterLanguage(HearingOptions hearingOptions, OverrideFields overrideFields, ReferenceDataServiceHolder referenceData, String adjournLanguage) throws InvalidMappingException {
 
         if (nonNull(overrideFields)
             && nonNull(overrideFields.getAppellantInterpreter())
             && nonNull(overrideFields.getAppellantInterpreter().getIsInterpreterWanted())) {
             return getOverrideInterpreterLanguage(overrideFields);
         }
 
+        if (nonNull(adjournLanguage)) {
```

Comment:
Shouldn't this also check if adjournment flag is enabled && adjournmentInProgress? Bc if either of those are false then this should be ignored.